FreeRTOS/Demo/CORTEX_M3_MPS2_QEMU_GCC/console.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * FreeRTOS V202011.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
This version should be updated since the 202012  release is now out

Reply:
done, checker wanted the old version